.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "submodules/eventuals"]
 	path = submodules/eventuals
 	url = https://github.com/3rdparty/eventuals.git
+    branch = main
```

Comment:
...Indentation 
...looks 
weird

Reply:
That's so odd, I thought I had fixed this. Thanks though!